### PR TITLE
GraphQl - Add storeLocale in StoreConfig

### LINKED ISSUE #2703: GraphQl - Add storeLocale in StoreConfig
Please add `storeLocale` configuration in `StoreConfig` type. 

I think the configuration should be retrieved from `\Adyen\Payment\Helper\Data::getStoreLocale()`

### PR DESCRIPTION
**Pull request by dimitriBouteille**
**Description**

This MR add `adyen_store_locale` in graphQl `StoreConfig` type based on `\Adyen\Payment\Helper\Data::getStoreLocale()`.

Fixes  #2703

### CODE REVIEW

**Comment by candemiralp** [comment]
Dear @dimitriBouteille,

Thank you for your contribution!

Best Regards,
Can

File: Model/Resolver/StoreConfig/StoreLocale.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+use Adyen\Payment\Helper\Data;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\GraphQl\Model\Query\Context;
+use Magento\Store\Api\Data\StoreInterface;
+use Reflet\GraphQlFaker\Attribute\FakeResolver;
+use Reflet\GraphQlFaker\Model\Resolver\NullResolver;
+
+class StoreLocale implements ResolverInterface
```
[the remark this answers is not in the document]
Comment (dimitriBouteille) [reply Model/Resolver/StoreConfig/StoreLocale.php:25]: I added a test, to see if it’s OK for you  :)

[the remark this answers is not in the document]
Comment (candemiralp) [reply Model/Resolver/StoreConfig/StoreLocale.php:25]: Thank you @dimitriBouteille, looks fantastic!

**Review by candemiralp — APPROVED** [review]

**Review by SushmitaThakur — APPROVED** [review]

File: view/frontend/web/js/model/adyen-payment-service.js
Comment (peterojo) [inline view/frontend/web/js/model/adyen-payment-service.js:1]: Hi @dimitriBouteille! 
Could you please clear the modifications in this file?

**Review by RokPopov — APPROVED** [review]

**Pull request merged**